patches/0062-Allow-DHCP-Broadcast-packets.patch

```diff
@@ -0,0 +1,74 @@
+From 52c1cff91bab2da916816783f53dc5ee47a9ee13 Mon Sep 17 00:00:00 2001
+From: Nagendrababu <nagendrababu.bonkuri@connectus.ai>
+Date: Thu, 9 Sep 2021 17:06:08 -0400
+Subject: [PATCH] WIFI-3905-AP does not forward dhcp broadcast offer
+
+This patch will add fix to allow DHCP broadcast packets, so that AP can
+forward DHCP broadcast offer to clients
+
+Signed-off-by: Nagendrababu <nagendrababu.bonkuri@connectus.ai>
+---
+ .../113-Allow-DHCP-broadcast-packet.patch     | 51 +++++++++++++++++++
+ 1 file changed, 51 insertions(+)
+ create mode 100644 target/linux/ipq807x/patches/113-Allow-DHCP-broadcast-packet.patch
+
+diff --git a/target/linux/ipq807x/patches/113-Allow-DHCP-broadcast-packet.patch b/target/linux/ipq807x/patches/113-Allow-DHCP-broadcast-packet.patch
+new file mode 100644
+index 0000000000..89110c55af
+--- /dev/null
++++ b/target/linux/ipq807x/patches/113-Allow-DHCP-broadcast-packet.patch
+@@ -0,0 +1,51 @@
++Index: linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/net/bridge/br_forward.c
++===================================================================
++--- linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce.orig/net/bridge/br_forward.c
+++++ linux-4.4.60-qsdk-10fd7d14853b7020b804acae690c8acec5d954ce/net/bridge/br_forward.c
++@@ -20,6 +20,37 @@
++ #include <linux/if_vlan.h>
++ #include <linux/netfilter_bridge.h>
++ #include "br_private.h"
+++#include <linux/ip.h>
+++#include <linux/udp.h>
+++
+++struct bootp_pkt {		/* BOOTP packet format */
```

Comment:
is there way to detect the dhcp packet without copying this whole structure here?

Reply:
To detect the DHCP packet there is local structure in the source code. And didn't find any global structure definitions to detect the DHCP packet.